Demo1.js

```diff
@@ -27,6 +27,15 @@ export default React.createClass({
     };
   },
 
+  handleTouchStart: function(key, pressLocation, event) {
+    this.handleMouseDown(key, pressLocation, event.touches[0]);
```

Comment:
Ah. And `touches[0]` specifically is one of these: https://developer.mozilla.org/en-US/docs/Web/API/Touch